modules/missionmail.js

```diff
@@ -0,0 +1,245 @@
+// @ts-check
+const u = require("../utils/utils");
+const c = require("../config/config.json");
+const receive = require("imapflow");
+const interpret = require("mailparser-mit");
+const htmlparse = require("html-to-text");
+const Augur = require("augurbot-ts");
+const { ButtonStyle } = require("discord.js");
+
+const [approveIdPrefix, rejectIdPrefix] = ["approveMissionMail", "rejectMissionMail"];
+
+const replyRegexes = [
+  // /^on[\s\n\r]+.+wrote:[\s\n\r]*$/im,
+  // /on[\s\n\r]+.+wrote:[\s\n\r]*/im,
+  /^on[^>]*@[^<]*wrote:\n\n>/im,
+  // /^on\s.+wrote:\s*$/im, :(){:|:&};:
+  // /On\s.+wrote:\s*/im,
+  /^>.*$/m,
+  /^-+ original message -+$/m,
+  /^from:.*$/m,
+  /^sent:.*$/m,
+  /^to:.*$/m,
+  /^subject:.*$/m,
+  /^date:.*$/m
+];
+
+const Module = new Augur.Module();
+
+async function loadEmails() {
+  const creds = c.google.mail;
+  try {
+    const receiver = new receive.ImapFlow({
+      auth: {
+        user: creds.email,
+        pass: creds.pass
+      },
+      host: 'imap.gmail.com',
+      port: 993,
+      logger: false
+    });
+
+    await receiver.connect();
+    await receiver.mailboxOpen("INBOX");
+
+    const sent = await sendUnsent(receiver);
+    receiver.close();
+
+    return sent;
+  } catch (error) {
+    u.errorHandler(error, "Loading Emails");
+  }
+}
+
+/**
+ * @param {receive.ImapFlow} receiver
+ * @returns {Promise<number>}
+ */
+async function sendUnsent(receiver) {
+  if (!receiver.usable) throw new Error("Missionary Email Receiver not usable, cannot check for new emails.");
+
+  const messageIds = await receiver.search({ or: u.db.sheets.missionaries.map((email) => ({ from: email })), seen: false });
```

Comment:
should see if theres an option to limit based on time. Getting emails older than a week should be unnecessary